dbt/adapters/databricks/connections.py

```diff
@@ -475,6 +476,17 @@ class DatabricksConnectionManager(SparkConnectionManager):
     TYPE: str = "databricks"
     credentials_provider: Optional[TCredentialProvider] = None
 
+    def cancel_open(self) -> List[str]:
+        from dbt.adapters.databricks.python_submissions import BaseDatabricksHelper
+
+        for run_id in BaseDatabricksHelper.run_ids:
+            logger.info(f"cancel run id {run_id}")
```

Comment:
I think this can be debug, and we should mention that it's a python model job.

Reply:
done!